src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2938,9 +2930,27 @@ private uint getJitFlags(ref CORJIT_FLAGS flags, uint sizeInBytes)
             return (uint)sizeof(CORJIT_FLAGS);
         }
 
+
+        InstructionSetFlags _actualInstructionSetSupported;
+        InstructionSetFlags _actualInstructionSetUnsupported;
```

Comment:
@davidwrighton I was digging through this over the weekend and this will always have the default value of 0. Should we actually set it somewhere?

Reply:
Sigh. Yes. Just below the !supportEnabled path should be setting this.